PrestaShop/PrestaShop
# [-] FO : Fix loyalty to work in 1.6.1.x

[-] FO : Fix loyalty to work in 1.6.1.4 (and probably earlier 1.6.1.x) with default-bootstrap theme
## Description

default bootstrap loyalty files needed updating to work. Noticed it was not working in 1.6.1.4.  The module must have gotten ahead of the theme module files.
## Steps to Test this Fix

1)  Install 1.6.1.4 with default-bootstrap.
2)  Notice loyalty does not work in the product page (says no points)
3)  Replace 2 files from the main module folder (/modules/loyalty) into default-bootstrap
4)  Notice it works.
## Forge Ticket (optional)

none
